ldes-server-infra-mongo/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/infra/mongo/entities/LdesFragmentEntity.java

```diff
@@ -10,7 +10,7 @@
 import java.util.List;
 
 @Document("ldesfragment")
-public class LdesFragmentEntity implements Comparable<LdesFragmentEntity> {
```

Comment:
not sure why you'd want to remove this?

Reply:
this requires the method compareTo to be overriden.
And according to java:S1210: "It is strongly recommended, but not strictly required that (x.compareTo(y)==0) == (x.equals(y))."

In the current implementation the compareTo method only uses the fragmentId
In our case we don't really that equality only depends on the fragmentId. (because that would mean that a fragment fragments can have different relations, members, ... and still be equal)

Hence, I removed the compareTo method, the Comparable interface and replaced the code usage of the compareTo with 
```
.min(Comparator.comparing(LdesFragment::getFragmentId))
```

Reply:
Well in theory we should have compared on the value of the path for a fragment. but the current approach is also fine for me now